packages/eas-cli/src/submit/utils/errors.ts

```diff
@@ -70,6 +71,8 @@ const SubmissionErrorMessages: Record<SubmissionErrorCode, string> = {
     'Invalid Provisioning Profile Signature (ITMS-90165)\n' +
     "Some of Apple's certificates have expired.\n" +
     'Please delete your Provisioning Profile from your account. Then rebuild the app interactively to generate a new one, and try submitting it to the App Store again.',
+  [SubmissionErrorCode.UPLOAD_TAKING_TOO_LONG_ERROR]:
+    'Fastlane run has reached the timeout limit. Please try again.',
```

Comment:
```suggestion
    'Submission has reached the timeout limit. Please try again.',
```
I'm not sure - do we want to inform users that we're using fastlane under the hood? 🤔